src/styles/stop-watch.scss

```diff
@@ -0,0 +1,66 @@
+.stop-watch {
+  display: flex;
+  justify-content: center;
+  &__container {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    width: 500px;
+
+    &:hover > * {
+      animation-play-state: paused;
+    }
+  }
+
+  &__center {
+    position: absolute;
+    z-index: 2;
+
+    width: 30px;
+    height: 30px;
+    background-color: rgb(101, 201, 231);
+    border-radius: 50%;
+
+  }
+
+  &__second {
+    position: absolute;
+    top: 10vh;
+
+    width: 10px;
+    height: 40vh;
+    background-color: rgb(101, 201, 231);
+    border-radius: 5px;
+
+    animation: circle 60s steps(60, end) infinite backwards;
+    transform-origin: bottom;
+  }
+
+  &__minute {
+    position: absolute;
+    top: 25vh;
+    z-index: 1;
+
+    width: 15px;
+    height: 25vh;
+    background-color: rgb(231, 101, 101);
+    border-radius: 5px;
+
+    animation: circle 3600s steps(60,end) infinite backwards;
+    transform-origin: bottom;
+  }
+}
+
+@import "./normalize.scss";
```

Comment:
you imported this file on main

Reply:
Thank you for your feedback, I've made changes))